modules/021-cni-cilium/hooks/set_cilium_mode_test.go

```diff
@@ -1,255 +0,0 @@
-/*
```

Comment:
Don't delete the test, need to test simple conditions.

Reply:
restored and modified